java/src/test/java/ai/onnxruntime/InferenceTest.java

```diff
@@ -1168,15 +1168,53 @@ public void testModelInputBOOL() throws OrtException {
         OrtSession session = env.createSession(modelPath, options)) {
       String inputName = session.getInputNames().iterator().next();
       Map<String, OnnxTensor> container = new HashMap<>();
+      long[] shape = new long[] {1, 5};
```

Comment:
>long[] shape = new long[] {1, 5}; [](http://example.com/codeflow?start=6&length=33)

Does Java allow
long[] shape = {1,5} ?

Reply:
It does, but that hides the object creation and we use explicit new calls everywhere else when making arrays.